test/e2e/network/service.go

```diff
@@ -2572,10 +2576,13 @@ var _ = common.SIGDescribe("Services", func() {
 		// assert 5 times that the first pause pod can connect to the Service locally and the second one errors with a timeout
 		serviceAddress := net.JoinHostPort(svc.Spec.ClusterIP, strconv.Itoa(servicePort))
 		for i := 0; i < 5; i++ {
-			// the first pause pod should be on the same node as the webserver, so it can connect to the local pod using clusterIP
+			// There's a terminating pod called webserver0 running on node0.
+			// There's a Service with internalTrafficPolicy=Local
+			// pausePod0 and pausePod1 are on node0 and node1 respectively.
+			// pausePod0 -> Service clusterIP succeeds because webserver0 is running on node0 and traffic policy is "Local"
+			// pausePod1 -> Service clusterIP fails because pausePod1 is on a different node and traffic policy is "Local"
```

Comment:
it seems more natural to say that the second connection fails because _webserver0_ is on a different node, not because _pausePod1_ is on a different node.

Reply:
updated, thanks